backend/LexBoxApi/Services/HgService.cs

```diff
@@ -36,14 +36,17 @@ public HgService(IOptions<HgConfig> options, IHttpClientFactory clientFactory, I
     [GeneratedRegex(Project.ProjectCodeRegex)]
     private static partial Regex ProjectCodeRegex();
 
+    public static string PrefixRepoRequestPath(string code) => $"{code[0]}/{code}";
```

Comment:
This will be fine once we forbid `-` as a starting character for project codes, but it's not a good idea to have `-` as a file or directory name in Linux.